Makefile

```diff
@@ -1,8 +1,13 @@
-isort:
-	isort --recursive
+ISORT=isort --recursive --skip .venv --skip .venvs --skip .tox
 
-flake8:
-	flake8 . --ignore=E501,E722 --count --statistics
+fmt:
+	$(ISORT)
+
+test:
+	$(ISORT) --check-only
+	flake8 .
+	mypy .
+	pytest
```

Comment:
@bobzoller `test` recipe should probably just run` tox` since these commands essentially mirror the ones in `tox.ini` and `tox` is what gets run by travi anyway.

Reply:
ah, I guess the only issue with that ^ is that tox will attempt to run the tests for all python versions unless you specify the `-e <python-version>` flag. I guess this is fine for now.

Reply:
yup, that's exactly why I ended up with what I have.  good question though.